packages/docs/src/lib/str.ts

```diff
@@ -0,0 +1,3 @@
+export function capitalize(str: string) {
+  return `${str.slice(0, 1).toUpperCase()}${str.slice(1, Infinity)}`;
```

Comment:
Why pass Infinity here?
```suggestion
  return `${str.slice(0, 1).toUpperCase()}${str.slice(1)}`;
```

Reply:
I didn't know that by omitting the second parameter it got it the rest, cool!